test/e2e/collection/namespace_filtering_test.go

```diff
@@ -0,0 +1,182 @@
+package collection
```

Comment:
Why did we need to completely rewrite this test instead of doing a `git mv` of the old to the new namespace and modify it accordingly

Reply:
@ajaygupta978 :point_up: